examples/js/controls/OrbitControls.js

```diff
@@ -933,7 +933,15 @@ THREE.OrbitControls = function ( object, domElement ) {
 	scope.domElement.addEventListener( 'touchend', onTouchEnd, false );
 	scope.domElement.addEventListener( 'touchmove', onTouchMove, false );
 
-	window.addEventListener( 'keydown', onKeyDown, false );
+	scope.domElement.addEventListener( 'keydown', onKeyDown, false );
+
+	// make sure element can receive keys.
```

Comment:
Maybe just leave this to the user instead, if this detail is blocking the PR? This should obviously be merged. In my opinion, the `domElement` argument should even be required at construction time.

Reply:
> the domElement argument should even be required at construction time (but that will be for another PR).

Related #16336. Notice that the PR was not accepted.

Reply:
But you closed it yourself without giving a reason.

Reply:
Look a the dates. If a PR is not accepted within two month, something is wrong with it.

Reply:
If a PR is not accepted within two months, _something is wrong_, but the PR may still be OK. It may have been forgotten/overlooked/down-prioritized by the people in charge. Judging by the comments, the reception was positive.